From 015dc7c37abd03be53b01baad67bf1a5f0bab364 Mon Sep 17 00:00:00 2001
From: ola.lilja <ola.lilja@intel.com>
Date: Thu, 7 Nov 2013 12:52:31 +0100
Subject: Adapt makefile to BT-mixin

BOARD_HAVE_BLUETOOTH_BCM is set to false when Broadcom is
not used.

Change-Id: I5190bbc55431939bb3bf0c51c7e5d165512730e3
Signed-off-by: ola.lilja <ola.lilja@intel.com>
---
 Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Android.mk b/Android.mk
index 2ab3cc6..8c8f711 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,6 +1,6 @@
 LOCAL_PATH := $(call my-dir)
 
-ifneq ($(BOARD_HAVE_BLUETOOTH_BCM),)
+ifeq ($(BOARD_HAVE_BLUETOOTH_BCM), true)
 
 include $(CLEAR_VARS)
 
-- 
1.7.4.1

